### ISSUE TITLE
Please tag a stable release

### ISSUE THREAD

**Issue by sjackman** [johnkerl/miller#12]
Thanks!

**Comment by johnkerl** [comment]
TL;DR is ... yes, absolutely. ;)

Longer version: I use manual CI (see also http://johnkerl.org/miller/doc/build.html) -- all unit/regression tests run on each build. So in some sense every commit is "stable". Moreover, https://github.com/johnkerl/miller/issues/8 asks for travis-ci which will simply automate what's already happening. I don't have any bugs I'm aware of right now, I use Miller daily & its feature set is pretty thorough for me at this point, and upcoming code-per-se work is just going to be addition of features (principally, RFC-compliant CSV I/O which is a big project).

So, in that sense the latest commit (https://github.com/johnkerl/miller/commit/d2dd5f976ed6cea865068b9cee59c0c2b206e410) is as stable as anything.

What *is* new since my release announcement on Saturday, though, is people compiling on other platforms than my usual two (both of which I use daily). Most of the open/closed issues are about minor compiler issues on other platforms, homebrew/deb setup, etc. This part is still in motion.

So if what you want is stable code for end use, use the above commit (or any other) & if it compiles on your *particular* platform, you're good to go. (And if not, please let me know.) Stable in terms of my being confident it's building on *all* platforms I know of, isn't there yet -- and I'll make a 1.0 tag per se at that point (hopefully in a week or two).

**Comment by sjackman** [comment]
My intention is simply to package Miller for Homebrew (see #1) and that can't happen until a stable release is tagged, per Homebrew's policy.

**Comment by johnkerl** [comment]
No worries. Tagging now ... :)

**Comment by johnkerl** [comment]
https://github.com/johnkerl/miller/releases

**Comment by sjackman** [comment]
Thanks, John!
